docs/source/model_design/vivarium_model_components/risk_factors/risk_effects/index.rst

```diff
@@ -102,38 +102,76 @@ as python code:
-        y = np.array(t3.values)  # df.draw_n
-        f_rr = scipy.interpolate.interp1d(x, y, kind='linear', bounds_error=False, fill_value=(min(y), max(y)))
-
-        # pick a tmrel between tmred.min and tmred.max --- for certain risk factors, the modeling team uploads a model for this, too --- shiny tool has information about this
-        # and calculate relative risk at tmrel
-        tmrel = np.random.uniform(tmred.min, tmred.max)
-        rr_at_tmrel = f_rr(tmrel)
-        t4 = t3 / rr_at_tmrel
-
-        # TODO: see if GBD would also benefit from clipping these RRs
-        t5 = np.clip(t4, 1.0, np.inf)
-
-        y = np.array(t5.values)
-        f_rr_2 = scipy.interpolate.interp1d(x, y, kind='linear', bounds_error=False, fill_value=(min(y), max(y)))
-        return f_rr_2
+  import numpy as np
+  import scipy.interpolate
+  import matplotlib.pyplot as plt
+  import gbd_mapping, vivarium_gbd_access
+
+  # Replace with your risk of interest
+  risk = gbd_mapping.risk_factors.high_systolic_blood_pressure # high_body_mass_index_in_adults
+  # Replace with your cause of interest
+  cause = gbd_mapping.causes.ischemic_heart_disease
+  age_group_id = 20 # 75 to 79
+  sex_id = 1 # Male
+  year_id = 2021
+
+  relative_risk_data = vivarium_gbd_access.gbd.get_relative_risk(
+      risk.gbd_id,
+      1, # Global
+      year_id=year_id,
+  )
+
+  # Subset to cause, age, and sex of interest
+  # If interested in multiple, would loop through them
+  relative_risk_data = relative_risk_data[
+      (relative_risk_data.cause_id == cause.gbd_id) &
+      (relative_risk_data.age_group_id == age_group_id) &
+      (relative_risk_data.sex_id == sex_id)
+  ].sort_values('exposure')
+
+  relative_risk_functions = {}
+
+  # Do calculation at the draw level
+  for draw_id in range(1_000):
+    relative_risk_draw = relative_risk_data[f'draw_{draw_id}']
+    # interpolate a continuous function between the points,
+    # and extrapolate outside the range with the endpoints
+    raw_relative_risk_function = scipy.interpolate.interp1d(
+        relative_risk_data.exposure,
+        relative_risk_draw,
+        kind='linear',
+        bounds_error=False,
+        fill_value=(
+            relative_risk_draw.min(),
+            relative_risk_draw.max(),
+        )
+    )
+
+    # pick a tmrel between tmred.min and tmred.max and calculate relative risk at tmrel
+    # for certain risk factors, the modeling team uploads a model for this with TMREL draws --
+    # those should be used instead of this, when available!
+    tmrel = np.random.uniform(risk.tmred.min, risk.tmred.max)
+    rr_at_tmrel = raw_relative_risk_function(tmrel)
+    normalized_relative_risk_draw = relative_risk_draw / rr_at_tmrel
+
+    # Clip relative risk to 1, since the TMREL is supposed to represent minimum risk
+    # NOTE: It is not clear if this is always appropriate, since TMRELs are across all
+    # risk-cause pairs involving this risk, not only the pair we are modeling
```

Comment:
@zmbc I didn't really know what this comment meant until you explained it in the doc session today. It would be good to put that explanation somewhere in the docs, but that should probably be a different PR.

Reply:
And again, if it's not clear whether this is always appropriate, then why do you think it's clear what GBD actually did?

Reply:
I've added a better comment here.